frontend/Dockerfile

```diff
@@ -1,13 +1,11 @@
-FROM node:20.7.0-alpine
+FROM node:20-alpine
```

Comment:
Ditto

Reply:
Details in this [comment](https://github.com/e-mission/nrel-openpath-join-page/pull/27#discussion_r1433245219).